### PR TITLE
Setup github actions

### PR DESCRIPTION
**Pull request by piotrekfus91**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+      - name: Build with Maven
+        run: mvn package
```
Comment (alien11689) [inline .github/workflows/build.yml]: maybe let's add mvn wrapper?

**Review by alien11689 — CHANGES_REQUESTED** [review]
If you want to go with github actions, let's get rid of travis file

File: .github/workflows/build.yml (hunk 1 above)
Comment (piotrekfus91) [reply .github/workflows/build.yml]: Ok, I added it in another PR: #16

Comment (alien11689) [reply .github/workflows/build.yml]: Remmber to use it here (and resolve)

File: mockserver-tests/src/test/groovy/pl/touk/mockserver/tests/MockServerHttpsTest.groovy [hunk 2]
```diff
@@ -15,13 +15,15 @@ import pl.touk.mockserver.client.RemoteMockServer
...
 import spock.lang.AutoCleanup
+import spock.lang.Ignore
 import spock.lang.Shared
 import spock.lang.Specification
 
 import javax.net.ssl.SSLContext
 import javax.net.ssl.SSLHandshakeException
 import java.security.KeyStore
 
+@Ignore('Upgrade of Java needed')
```
Comment (alien11689) [inline mockserver-tests/src/test/groovy/pl/touk/mockserver/tests/MockServerHttpsTest.groovy:26]: Let's add an issue for that

**Review by alien11689 — APPROVED** [review]

File: mockserver-tests/src/test/groovy/pl/touk/mockserver/tests/MockServerHttpsTest.groovy (hunk 2 above)
Comment (piotrekfus91) [reply mockserver-tests/src/test/groovy/pl/touk/mockserver/tests/MockServerHttpsTest.groovy:26]: Done

**Pull request merged**